zerver/openapi/zulip.yaml

```diff
@@ -1210,6 +1210,15 @@ paths:
   /users/me/subscriptions:
     get:
       description: Get information on every stream the user is subscribed to.
+      parameters:
+      - name: include_subscribers
+        in: query
+        description: Set to `true` if you would like each stream's info to include a list of
+          current subscribers to that stream. (This may be significantly slower.)
```

Comment:
(This is significantly slower in organizations with thousands of users)

Reply:
Is it actually determined by the number of users in the organization, or only by the number of users in the subscribed channels?

Reply:
It's driven by total number of users in the subscribed streams; they're often pretty correlated though; if you don't have thousands of people in the organization, you can't get thousands in a stream :).